.travis.yml

```diff
@@ -1,11 +1,28 @@
+# https://docs.travis-ci.com/user/languages/javascript-with-nodejs/
 language: node_js
 node_js:
   - 'lts/*'
 cache: yarn
+env:
+  global:
+  - ENCRYPTED_VARIABLE_TOKEN: "a36f85be0be9"
+  - COMMIT_AUTHOR_EMAIL: "kubevirt-builder@redhat.com"
+  - COMMIT_AUTHOR_NAME: "KubeVirt Builder"
+notifications:
+  email: false
 before_install:
+  # install and use the latest version of Yarn
   - curl -o- -L https://yarnpkg.com/install.sh | bash
-  - export PATH="$HOME/.yarn/bin:$PATH"
+  - export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
+install:
   - yarn install
+before_script:
+  - node tools/travis/beforeScript.js
 script:
   - yarn build
   - yarn coveralls
+after_success:
+  - node tools/travis/afterSuccess.js
+branches:
+  only:
```

Comment:
Can we keep testing on open/force=pushed PRs?
If done in a follow-up, let's have it prepared already before merging this. It would block other PRs otherwise ...

Reply:
This should work for our recent workflow. I think we can adjust later, if ever needed:

https://stackoverflow.com/questions/31882306/how-to-configure-travis-ci-to-build-pull-requests-merges-to-master-w-o-redunda